examples/http/http-producer.yaml

```diff
@@ -32,7 +32,7 @@ spec:
               value: "100"
             - name: LOG_LEVEL
               value: DEBUG
-          image: quay.io/strimzi-test-clients/test-client-http-producer:latest-kafka-3.0.0
+          image: quay.io/strimzi-test-clients/test-client-http-producer:latest
```

Comment:
Why is `latest` used there?

Reply:
On the `main`, we are using the `latest` tag. For releases it is changed to the release tag - for example here: https://github.com/strimzi/test-clients/blob/0.1.1/examples/kafka/kafka-consumer.yaml

Reply:
Thanks